source/sdr.rst

```diff
@@ -46,7 +46,7 @@ The amount of annual soil loss on pixel :math:`i`, :math:`usle_i` (units: :math:
 
 where
 
- * :math:`R_i` is rainfall erosivity (units: :math:`MJ\cdot mm (ha\cdot hr)^{-1})`,
+ * :math:`R_i` is rainfall erosivity (units: :math:`MJ\cdot mm (ha\cdot hr\cdot yr)^{-1})`,
 
  * :math:`K_i` is soil erodibility (units: :math:`ton\cdot ha\cdot hr (MJ\cdot ha\cdot mm)^{-1}`),

```

Comment:
Are the units of erodibility correct _(hours_ versus _years_)? Why is _ha_ in both the denominator and the numerator when they would cancel out?

Reply:
@jagoldstein, the units for erosivity are weird. As I commented previously: The correct units are listed in the Data Needs section: (MJ-mm)/(hour-ha-year). Also see this document, which walks through the gnarly units involved in the USLE: https://www.engr.colostate.edu/~pierre/ce_old/Projects/linkfiles/USLE%20Unit%20conversions.pdf.

Reply:
Thanks, yeah the units get weird for both erodibility and erosivity, and I see different variations across different sources. I was specifically checking on _soil erodibility_ units here, which Emily did not change the definition of yet, but I just wanted to confirm that what's written for that is accurate.

Reply:
It's strange that the _ha_s don't cancel, but that paper @newtpatrol linked says:
> Although hectares  in  the numerator cancels hectares in the denominator,  both  are left  to show that  K  is  soil loss  per  unit area per unit of  E I .   
It also shows _hours_ for erodibility (see the table on page 3)